test/keyboard.html

```diff
@@ -284,11 +284,12 @@
       });
 
       it('escape key event should not be propagated', () => {
```

Comment:
Let's add a test to verify that the escape key event gets propagated if the dropdown is closed.